### DISCUSSION TITLE
Window Position on Startup

### DISCUSSION THREAD

**Discussion by manuelc2209** [Rem0o/FanControl.Releases#2445] [Q&A]
Hi guys,

Not sure if this is something related with the App itself, I have a multi monitor setup, and one of those monitors, contains the Fan Control app _(Which btw, excellent app)_. The only thing that is annoying to me is that the window always spawns up on a random position with some offset and not where i left it last time (Is this something we can do here? I'm happy to try & help too) or this related to OS itself and the way it saves the App's positions?

Thanks!

**Comment by Faggi11** [comment]
Hey, if you are using W11, there is an option in the settings, where you can "Let Windows knows the last window-position"!
Using W10 or so, use Powertoys from MS, ............ but if you switch the monitors, it could be mismatched.

**Reply by manuelc2209** [reply]
Yeah i know that mate, all my other tools are correctly positioned, the only one isn't is Fan Control itself unfortunately, might be a Win11 issue somehow, if someone can provide more details on how it works, that would be cool!

**Reply by Faggi11** [reply]
Get a try with PowerToys (inside-app called "FancyZones").... i use that, but for some other circumstances which will work for me (switch from a Single WQHD-Monitor to a 3x FHD-Monitor
